Source/missiles.cpp

```diff
@@ -251,7 +251,7 @@ bool MonsterMHit(int pnum, int monsterId, int mindam, int maxdam, int dist, miss
 	} else {
 		if (monster.mode != MonsterMode::Petrified && MissilesData[t].mType == 0 && HasAnyOf(player._pIFlags, ItemSpecialEffect::Knockback))
 			M_GetKnockback(monster);
-		if (monster.type().type != MT_GOLEM)
+		if (!monster.isPlayerOwned())

```

Comment:
I'm assuming there are no non-player golems, might need to revert this one so they're still immune to knockback

Reply:
> I'm assuming there are no non-player golems

correct